src/util/Bundles.ts

```diff
@@ -27,8 +27,8 @@ declare type i18nOptions = {
 }
 
 declare type Assets = {
```

Comment:
is this change a must have? we should avoid changes like these, since it affects types for v2

Reply:
I can totally do this, will update. Thank you!